build.gradle

```diff
@@ -255,6 +255,7 @@ configure(javaProjects) { subproject ->
 		configurations.all {
 			resolutionStrategy.cacheChangingModulesFor 60, 'minutes'
 			exclude group: 'ch.qos.logback', module: 'logback-classic'
+			exclude group: 'org.projectreactor', module: 'reactor-core'
```

Comment:
Unfortunately we can do this.
`GatewayProxyFactoryBean#setReactorEnvironment(Environment reactorEnvironment)`
And SF tries to do some type extraction from similar public methods on bean definition phase.

To overcome that and really allow to have `Reactor` as `optional` we did this in SI-4.2:
```
setReactorEnvironment(Object reactorEnvironment)
```